tfplan2cai/test/init_test.go

```diff
@@ -59,6 +59,12 @@ func init() {
 		log.Printf("Missing required env var TEST_ANCESTRY. Default (%s) will be used.", defaultAncestry)
 		ancestry = defaultAncestry
 	}
+
+	if _, ok := os.LookupEnv("GOOGLE_APPLICATION_CREDENTIALS"); credentials != "" && !ok {
+		log.Printf("Setting GOOGLE_APPLICATION_CREDENTIALS with credentials")
+		os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", data.Provider["credentials"])
```

Comment:
`data.Provider["credentials"]` isn't set until later in this function, so it can't be accessed here. You could use `credentials` directly instead.

I think we might also want to _always_ override GOOGLE_APPLICATION_CREDENTIALS with the value from `credentials` - for example, if TEST_CREDENTIALS was set but GOOGLE_APPLICATION_CREDENTIALS was not.

Could you also add a comment to the code explaining why we need to do this?

Reply:
Turns out this part is not necessary, so removed it.